mkdocs.yml

```diff
@@ -60,61 +59,74 @@ markdown_extensions:
       use_pygments: true
   - pymdownx.superfences
   - pymdownx.emoji:
-      emoji_index: !!python/name:materialx.emoji.twemoji
-      emoji_generator: !!python/name:materialx.emoji.to_svg
+      emoji_index: !!python/name:material.extensions.emoji.twemoji
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
 
 nav:
   - Home: index.md
+  - 'Board' :
```

Comment:
why did the sorting change?

Reply:
To reiterate the file was not manually edited. It was generated 'python3 tools/mkArmbianDocs.py'

Reply:
The code that does this is Lines 144 - 147 https://github.com/armbian/documentation/blob/d49213ebfb6de8273b145a00b79d5f93c0f4c917/tools/mkArmbianDocs.py#L144